scripts/run_action.mjs

```diff
@@ -0,0 +1,90 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import url from "url";
+
+const resolveActionPath = async actionPath => {
+  let result = "";
+
+  if (!actionPath) return result;
+
+  const currentDirectory = (await $`pwd`).toString().trim();
```

Comment:
note to self: replace with `process.cwd()` or something

Reply:
Yes, let's replace in this PR